<?xml version="1.0"?>
<launch>

	<arg name="controller" default="1"/>

	<node pkg="turtlesim" name="turtlesim" type="turtlesim_node" output="screen"/>

	<node pkg="turtle_controller" name="interface" type="turtle_controller_interface" output="screen">
		<rosparam command="load"
			file="$(find turtle_controller)/config/interface.yaml" />
	</node>

	<node pkg="turtle_controller" name="turtle_controller" type="turtle_controller" output="screen">
		<param name="controller_alg" type="int" value="$(arg controller)"/>
		<rosparam command="load"
			file="$(find turtle_controller)/config/config.yaml" />
	</node>


</launch>
